specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

```diff
@@ -4408,6 +4499,93 @@
         }
       },
       "description": "The result of the List migrationConfigurations operation."
+    },
+    "Subnet": {
```

Comment:
>" [](start = 4, length = 1)

"required" properties not specified for the new types

Reply:
added required property